pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -2331,7 +2333,287 @@ def ReuseObjectiveFunctionRule(model):
+            return model.v_Z == (
+                model.v_C_TotalSourced
+                + model.v_C_TotalDisposal
+                + model.v_C_TotalTreatment
+                + model.v_C_TotalReuse
+                + model.v_C_TotalPiping
+                + model.v_C_TotalStorage
+                + model.v_C_TotalTrucking
+                + model.v_C_TreatmentOpex_surrogate
+                + model.v_C_TreatmentCapEx_surrogate
+                + model.p_alpha_AnnualizationRate
+                * (
+                    model.v_C_DisposalCapEx
+                    + model.v_C_StorageCapEx
+                    + model.v_C_PipelineCapEx
+                    + model.v_C_TreatmentCapEx
+                )
+                + model.v_C_Slack
+                - model.v_R_TotalStorage
+                # + 45*model.vb_y_bin
+            )
+
+        model.CostSurrogateObjectiveFunctionRule = Constraint(
+            rule=CostSurrogateObjectiveFunctionRule, doc="Cost objective function"
+        )
+
+        # Define constraints #
+        model.inlet_salinity.fix(salinity_dict["inlet_salinity"])
+        model.recovery.fix(salinity_dict["recovery"])
+        model.surrogate_costs = SurrogateBlock(model.s_R, model.s_T)
+        model.model_units["L_per_s"] = pyunits.L / pyunits.s
+        conversion_factor = pyunits.convert_value(
+            1,
+            from_units=model.model_units["volume_time"],
+            to_units=model.model_units["L_per_s"],
+        )
+
+        model.v_T_Treatment_scaled = Var(
+            model.s_R,
+            model.s_T,
+            within=NonNegativeReals,
+            initialize=1 * 5 * conversion_factor,
+        )
+
+        cap_lower_bound, cap_upper_bound = (
+            0 * 7 * conversion_factor,
+            100 * 7 * conversion_factor,
+        )
+
+        for i in model.s_R:
+            for t in model.s_T:
+                if model.p_chi_DesalinationSites[i]:
+                    model.v_T_Treatment_scaled[i, t].setlb(cap_lower_bound)
+                    model.v_T_Treatment_scaled[i, t].setub(cap_upper_bound)
+
+                else:
+
+                    model.v_T_Treatment_scaled[i, t].fix(0)
+
+        def scalingTreatment(model, r, t):
+            if model.p_chi_DesalinationSites[r]:
+                return model.v_T_Treatment_scaled[r, t] == conversion_factor * (
+                    sum(
+                        model.v_F_Piped[l, r, t]
+                        for l in model.s_L
+                        if (l, r) in model.s_LLA
+                    )
+                    + sum(
+                        model.v_F_Trucked[l, r, t]
+                        for l in model.s_L
+                        if (l, r) in model.s_LLT
+                    )
+                )
+            else:
+                return Constraint.Skip
+
+        model.treatment_vol = Constraint(model.s_R, model.s_T, rule=scalingTreatment)
+
+        keras_surrogate = KerasSurrogate.load_from_folder("md_keras")
```

Comment:
Please update the configuration to include options for running with either md_keras or mvc_keras. Additionally, for subsequent updates, consider implementing the capability to run both simultaneously, using distinct binaries for each.